app/routes/action.js

```diff
@@ -18,6 +18,22 @@ const getActionsForLandUses = (landUseCodes) => {
   })
 }
 
+const checkAreaAppliedForValid = (userSelectedActions, landParcel) => {
+  let message
+  for (const action of userSelectedActions) {
+    const areaAppliedFor = parseFloat(action.quantity)
+    const area = parseFloat(landParcel.area)
```

Comment:
This is a constant within the for loop so should be hoisted out of it

Reply:
Done 👍